.github/workflows/ci-terraform.yml

```diff
@@ -0,0 +1,97 @@
+name: Terraform validation and linting
+on:
+  push:
+    paths: ['**/*.tf', '**/*.hcl', '.github/workflows/ci-terraform.yml']
+env:
+  TF_IN_AUTOMATION: true
+  TF_PLUGIN_CACHE_DIR: ${{ github.workspace }}/.terraform.d/plugin-cache
+jobs:
+  tflint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout source code
```

Comment:
maybe checkout the latest commit only, i.e.:
with:
          fetch-depth: 0

Reply:
That actually causes it to fetch all revisions. The default is `fetch-depth: 1`, which fetches only the latest revision. See https://github.com/actions/checkout#checkout-v2

Reply:
thanks, I got confused on that one.